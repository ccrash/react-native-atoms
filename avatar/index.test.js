import React from 'react'
import renderer from 'react-test-renderer'

import Avatar from './index'

it('Renders an avatar from Name using capital letters', () => {
  expect(renderer.create(<Avatar name={'Silk Fred'} />)).toMatchSnapshot()
})

// it('Renders an avatar from Image', () => {
//   expect(renderer.create(<Avatar src={Images.defaultAvatar} />)).toMatchSnapshot()
// })
