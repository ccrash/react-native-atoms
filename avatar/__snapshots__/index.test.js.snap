// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders an avatar from Name using capital letters 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#222",
        "borderColor": "#000000",
        "borderRadius": 50,
        "borderWidth": 1,
        "height": 40,
        "justifyContent": "center",
        "width": 40,
      },
      undefined,
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#FFFFFF",
        },
        undefined,
      ]
    }
  >
    S F
  </Text>
</View>
`;
